examples/ExampleApp.py

```diff
@@ -249,36 +256,78 @@ def match_multiline(self, text, delimiter, in_state, style):
                 length = len(text) - start + add
             # Apply formatting
             self.setFormat(start, length, self.styles[style])
+            # Highlighting sits on top of this formatting
             # Look for the next match
             match = delimiter.match(text, start + length)
             start = match.capturedStart()
 
+        self.applySearchHighlight(text)
+
         # Return True if still inside a multi-line string, False otherwise
         if self.currentBlockState() == in_state:
             return True
         else:
             return False
 
+    def applySearchHighlight(self, text):
+        if not self.searchText:
+            return
+        expr = f'(?i){self.searchText}'
+        style = charFormat('#000', background='y')
+        for match in re.finditer(expr, text):
+            start = match.start()
+            length = match.end() - start
+            self.setFormat(start, length, style)
+
+
+def unnestDict(exDict):
+    """Converts a dict-of-dicts to a singly nested dict for non-recursive parsing"""
+    out = {}
+    for kk, vv in exDict.items():
+        if isinstance(vv, dict):
+            out.update(unnestDict(vv))
+        else:
+            out[kk] = vv
+    return out
+
 
 
-class ExampleLoader(QtGui.QMainWindow):
+class ExampleLoader(QtWidgets.QMainWindow):
     def __init__(self):
-        QtGui.QMainWindow.__init__(self)
+        QtWidgets.QMainWindow.__init__(self)
         self.ui = ui_template.Ui_Form()
-        self.cw = QtGui.QWidget()
+        self.cw = QtWidgets.QWidget()
         self.setCentralWidget(self.cw)
         self.ui.setupUi(self.cw)
         self.setWindowTitle("PyQtGraph Examples")
-        self.codeBtn = QtGui.QPushButton('Run Edited Code')
-        self.codeLayout = QtGui.QGridLayout()
+        self.codeBtn = QtWidgets.QPushButton('Run Edited Code')
+        self.codeLayout = QtWidgets.QGridLayout()
         self.ui.codeView.setLayout(self.codeLayout)
         self.hl = PythonHighlighter(self.ui.codeView.document())
-        app = QtGui.QApplication.instance()
+        app = QtWidgets.QApplication.instance()
         app.paletteChanged.connect(self.updateTheme)
-        self.codeLayout.addItem(QtGui.QSpacerItem(100,100,QtGui.QSizePolicy.Policy.Expanding,QtGui.QSizePolicy.Policy.Expanding), 0, 0)
```

Comment:
This is the change that's causing the enum issue.

Reply:
Right, is that the same output you get when you run your uic generator?